/**:
  ros__parameters:
    image_topic: "/cam0/image_raw"
    imu_topic: "/imu0"
    weight_file: "/weights/superpoint_lightglue_pipeline_240_320_1024.engine"
    imu_frame: "imu4"
    camera_frame: "cam0"
    base_frame: "base_link"
    odom_frame: "odom"
    map_frame: "map"
    resize_width: 320
    resize_height: 240
    num_keypoints: 1024 #According to the NN model
    fx: 320.0
    fy: 320.0
    cx: 320.0
    cy: 240.0
    absolute_coords: false
    pose_initializer: "local"
    # Reference needs to be provided if pose initializer is gnss
    # Reference is the LLA of map frame
    reference_lat: 0.0
    reference_lon: 0.0
    reference_alt: 0.0
